services/stream-eventbridge-ms/lambdas/streamProcessor.js

```diff
@@ -0,0 +1,30 @@
+import AWS from 'aws-sdk';
+
+const eventBridge = new AWS.EventBridge();
+
+/**
+ * Lambda function takes DynamoDB stream events and
+ * publishes them to an EventBridge Event Bus
+ */
+export const main = async event => {
+  const eventsToPut = [];
+
+  for (const record of event.Records) {
+    console.log(`Event: ${record.eventName}/${record.eventID}`);
+
+    const [tableArn] = record.eventSourceARN.split('/stream');
+
+    eventsToPut.push({
+      Time: record.dynamodb.ApproximateCreationDateTime,
+      Source: 'my-service.database',
```

Comment:
is this correct? my-service, feels like boilerplate code :D

Reply:
Correct!